### ISSUE TITLE
Hitting [xcode] ld: library not found for -lGooglePlayGames

### ISSUE THREAD

**Issue by mattr-swol** [playgameservices/play-games-plugin-for-unity#1994]
Hello,
We have ran into this build issue on xcode and have set NO_GPGS flag in the define symbols for Unity.

Has anyone encountered this and or/has solution ?
Thanks

**Comment by retNull** [comment]
Find GPGS_Dependencies.xml and remove:

```xml
<!-- iOS Cocoapod dependencies can be specified by each iosPod element. -->
  <iosPods>
    <!-- iosPod supports the following attributes:
         * "name" (required)
           Name of the Cocoapod.
         * "version" (optional)
           Cocoapod version specification for the named pod.
           If this is not specified the latest version is used.
         * "bitcodeEnabled" (optional)
           Whether this Cocoapod requires bitcode to be enabled in Unity's
           generated Xcode project.  This is "true" by default.
         * "minTargetSdk" (optional)
           The minimum iOS SDK required by this Cocoapod. -->
    <iosPod name="GooglePlayGames" version="5.1.2" bitcodeEnabled="false"
            minTargetSdk="6.0">
      <!-- Set of source URIs to search for this Cocoapod spec.
           By default Cocoapods will attempt to fetch the pod specs from:
           * $HOME/.cocoapods/repos
           * https://github.com/CocoaPods/Specs
           -->
      <sources>
        <source>https://github.com/CocoaPods/Specs</source>
      </sources>
    </iosPod>
  </iosPods>
```

**Comment by claywilkinson** [comment]
Thanks @retNull for the solution.  Feel free to submit a pull request commenting out or removing this section.  NO_GPGS is intended to remove all GPGS code and dependencies from the project.  Just as a reminder, iOS support is deprecated and will be removed soon: https://android-developers.googleblog.com/2017/04/focusing-our-google-play-games-services.html

**Comment by SlobodanNikolic** [comment]
Hi, I dont have GPGS_Dependencies.xml file anywhere. Does someone have another solution? Thanks

**Comment by claywilkinson** [comment]
This is fixed in 0.9.50, there is no more iOS code added from this plugin when building for iOS.
